tests/testsuite/warn_on_failure.rs

```diff
@@ -95,7 +95,6 @@ fn no_warning_on_bin_failure() {
 [COMPILING] foo v0.0.1 ([ROOT]/foo)
 error[E0425]: cannot find function `hi` in this scope
 ...
-For more information about this error, try `rustc --explain E0425`.
```

Comment:
Could you squash this into the relevant commit?

Reply:
Done.